index.html

```diff
@@ -0,0 +1,396 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      youtube-page
+    </title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="header">
+      <div class="right-section">
+        <img class="hamburger-menu" src="header-icons/hamburger-menu.svg" alt="">
+       <div class="youtube-logo-container">
+        <img class="youtube-logo"src="header-icons/youtube-logo.svg" alt="">
```

Comment:
Kindly add a relevant alternative (alt="relevant name to the image")name to the img element

Reply:
I think it would be good if you added relevant alternatives names to the img tag. This will be very helpful for users using screen readers